testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/web/persistence/AbstractDatabasePersistenceWebFailoverTestCase.java

```diff
@@ -73,7 +73,7 @@ public static void afterClass() throws Exception {
     public static class ServerSetupTask extends CLIServerSetupTask {
         public ServerSetupTask() {
             this.builder.node(THREE_NODES)
-                    .setup("/subsystem=datasources/data-source=web-sessions-ds:add(jndi-name=\"java:jboss/datasources/web-sessions-ds\", enabled=true, use-java-context=true, connection-url=\"jdbc:h2:tcp://localhost:%s/./web-sessions\", driver-name=h2", DB_PORT)
+                    .setup("/subsystem=datasources/data-source=web-sessions-ds:add(jndi-name=\"java:jboss/datasources/web-sessions-ds\", enabled=true, use-java-context=true, connection-url=\"jdbc:h2:tcp://localhost:%s/./web-sessions;VARIABLE_BINARY=TRUE\", driver-name=h2", DB_PORT)
```

Comment:
@pferraro @rhusar FYI re this. See the PR description or first commit's message for more about this.

Reply:
👍